cmake/modules/FindPySpark.cmake

```diff
@@ -0,0 +1,74 @@
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+#
+# Try to find pyspark
+# Once done this will define
+#
+#  PYSPARK_FOUND - system has pyspark and its dependencies are met
+#
+
+message(STATUS "Looking for pyspark dependency: Java")
+find_package(Java 1.8 COMPONENTS Runtime)
```

Comment:
Did you test distributed RDataframe + pyspark with both Java 1.8 and 1.9 >? (just in case checking)

Reply:
It has been tested with Java 1.8 and Java 11, which are the default respectively for Spark 2.4 (LTS of Spark 2.x) and Spark 3.x